scenarioo-client/test/protractorE2E/specs/show_step.js

```diff
@@ -146,10 +150,101 @@ useCase('Show step')
         scenario('Step without HTML source attached')
             .description('If the step data contains no html source data, the HTML tab should not be displayed at all')
             .it(function () {
-
                 stepPage.goToPage('/step/Donate/find_donate_page/startSearch.jsp/0/0');
                 stepPage.assertHtmlTabIsHidden();
                 step('A step with no HTML source attached');
             });
 
+
+        scenario('ScenarioPage with comparisons')
```

Comment:
what does a test that tests the scenario overview page (showing all steps) to do with use case "Show Step" ? and it looks confusing to me that the `stepPage` seems now to contain functionality to test also the scenario overview??